lib/helper/FileSystem.js

```diff
@@ -60,6 +60,26 @@ class FileSystem extends Helper {
     assert.ok(fileExists(this.file), `File ${name} not found in ${this.dir}`);
   }
 
+  /**
+   * Waits for file to be present in current directory.
+   *
+   * ```js
+   * I.handleDownloads();
+   * I.click('Download large File');
+   * I.amInPath('output/downloads');
+   * I.waitForFile('largeFilesName.txt');
+   * ```
+   * @param {string} name
+   * @param {?number} [sec=1] seconds to wait
+   */
+  async waitForFile(name, sec = 1) {
+    const timeout = sec ? sec * 1000 : 1000;
+    this.file = path.join(this.dir, name);
+    this.debugSection('File', this.file);
+    await checkExistsWithTimeout(this.file, timeout);
+    assert.ok(fileExists(this.file), `File ${name} not found in ${this.dir}`);
```

Comment:
So, this function is waiting for file then we basically return a boolean. The assertion can be done later. I guess it’s better.